tools/testing/e2e/iree-e2e-conv2d-test.cc

```diff
@@ -9,6 +9,8 @@
 #include <stdlib.h>
 #include <string.h>
 
+#include <stdexcept>
```

Comment:
we don't use exceptions in runtime code so this should not be required

Reply:
Yes, this is irrelevant. I used it for my local debugging, and am cleaning the code again.